content/pages/education/eligibility.md

```diff
@@ -79,8 +83,7 @@ Also, if you haven’t used all of your Post-9/11 GI Bill benefits, you may be a
 
 ### Ready to apply?
```

Comment:
This may need to be removed here as it is now added dynamically.

Reply:
I'm not sure what you mean.

Reply:
@peggygannon [asked](https://github.com/department-of-veterans-affairs/vets-website/pull/6642#pullrequestreview-70914527) that Ready to apply? not be displayed if there is a saved form in progress card, perhaps that is not the intention here however.

Reply:
@cehsu @jbalboni hi, jeff and claire, yes, when an "in progress form" message appears I had asked that the "Ready to apply?" header be removed, as it's a confusing UX for users to see both.